pkg/where/skinny.go

```diff
@@ -0,0 +1,123 @@
+package where
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/lib/pq"
+)
+
+// The where clause for queries against the geo_metrics table looks like this:
+//
+// WHERE (
+//	   geography_code IN ( ... )
+//     OR
+//     geography_code BETWEEN (low, high)
+//	   ...
+// ) AND (
+//	   category_code IN ( ... )
+//     OR
+//     category_code BETWEEN (low, high)
+//     ...
+// )
+//
+// Because of the size of the geo_metrics table, we require at least one geography_code
+// and at least one category_code to be specified.
+//
+//
+
+// SkinnyWhere generates the WHERE clause for queries against the geo_metrics table.
+// rows and cols are the multivalue rows= and cols= arguments.
+//
+func SkinnyWhere(rows, cols []string) (string, error) {
```

Comment:
is 'vertically mixed table' better than 'skinny table'?

Reply:
I think we can rationalise the naming as we add different queries. Definitely needs work.